### PR TITLE
Add dev.rzia.minesector

### PR DESCRIPTION
**Pull request by grassdne**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: SDL2_ttf.yml
Comment (hfiguiere) [inline SDL2_ttf.yml:1]: 2.20.0 is in the runtime. isn't it enough?

File: SDL2_mixer.yml
Comment (hfiguiere) [inline SDL2_mixer.yml:1]: 2.6.3 is in the runtime

File: SDL2_image.yml
Comment (hfiguiere) [inline SDL2_image.yml:1]: 2.6.3 is in the runtime.

File: dev.rzia.minesector.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+modules:
+  - shared-modules/SDL2/SDL2-with-libdecor.json
+  - SDL2_ttf.yml
+  - SDL2_mixer.yml
+  - SDL2_image.yml
+  - name: build
+    buildsystem: cmake
```
Comment (hfiguiere) [inline dev.rzia.minesector.yml:20]: does it build with `cmake-ninja`?

**Comment by hfiguiere** [comment]
bot, build dev.rzia.minesector

**Comment by grassdne** [comment]
all the sdl2 stuff is in the freedesktop runtime? that's awesome!

**Comment by grassdne** [comment]
Yes, yes, yes, and yes!
It all works!
Flatpak is amazing! You're amazing!

File: dev.rzia.minesector.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+rename-desktop-file: minesector.desktop
+rename-icon: minesector
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+modules:
+  - name: build
```
Comment (hfiguiere) [inline dev.rzia.minesector.yml:15]:
```suggestion
  - name: minesector
```

**Comment by hfiguiere** [comment]
bot, build dev.rzia.minesector

**Pull request closed without merging**